core/trino-main/src/main/java/io/trino/execution/DistributionSnapshot.java

```diff
@@ -110,15 +110,15 @@ public DistributionSnapshot(Distribution<?> distribution)
                 distribution.getTotal(),
                 distribution.getMin(),
                 distribution.getMax(),
-                distribution.getPercentile(0.01),
-                distribution.getPercentile(0.05),
-                distribution.getPercentile(0.10),
-                distribution.getPercentile(0.25),
-                distribution.getPercentile(0.50),
-                distribution.getPercentile(0.75),
-                distribution.getPercentile(0.90),
-                distribution.getPercentile(0.95),
-                distribution.getPercentile(0.99));
+                distribution.getPercentile(1),
```

Comment:
How about using`TDigestHistogram#getP01`, `TDigestHistogram#getP05` and so forth?

Reply:
We get `Distribution` as input and it has just:
![image](https://github.com/user-attachments/assets/3b0488d8-73ea-421c-a072-b7b4e9486f14)